### DISCUSSION TITLE
Missing access to benefits.

### DISCUSSION THREAD

**Discussion by m4wx3** [community/community#140687] [GitHub Education]
### Hi everyone,

Hey everyone! Its been around 4ish days coming up on 5 since i've been approved for the Student Developer pack & i still haven't gotten access to my benefits.. If anyone can help with like why this is happening would be appreciated!

**Comment by davevad93** [comment]
Hi @R-41N ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @R-41N was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
